### PR TITLE
14.0 auto resize jke

### PR DESCRIPTION
**Pull request by JKE-be**
Resize by default, similarly to Whatsapp, all ir.attachment picture to avoid big image.

An expense report in 10M px is useless ;) 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by odony** [comment]
@robodoo  override=ci/security
Rationale: false positives with `Image.open(bytes)`, and safe use of `os.path` in test code.

**Comment by JKE-be** [comment]
@robodoo r+

File: odoo/tools/image.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def image_base64(self, quality=0, output_format=''):
             output_format = 'JPEG'
 
         if not self.operationsCount and output_format == self.original_format and not quality:
-            return self.base64_source
+            return self.image
```
Comment (seb-odoo) [inline odoo/tools/image.py:130]: This diff is not correct, `self.image` is the PIL class, not the bytes, and the method is supposed to return the original bytes.

Error from the caller when reaching this case:
`TypeError: a bytes-like object is required, not 'PngImageFile'`

**Review by seb-odoo — COMMENTED** [review]
Hello, small remark because we tried to use this new method to get rid of base64 in a new branch and we got some issues with this PR.

**Pull request closed without merging**